### PR TITLE
New Languages of India Cheat Sheet

### PR DESCRIPTION
**Pull request by sinwar**
What does your Instant Answer do?
My Instant Answer tells the official language of each state of India.This is the cheat sheet for languages of indian states. There are different official language of each state in india so this cheat sheet is created.

What is the data source for your Instant Answer? (Provide a link if possible)
Source for my instant answer is Wikipedia- https://en.wikipedia.org/wiki/Languages_with_official_status_in_India

Why did you choose this data source?
Wikipedia is perfect for this having updated information.

Are there any other alternative (better) data sources?
Nope, Not updated

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
People who want to gain general knowledge. There are plenty of number in india of this type.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
yup

Are you having any problems? Do you need our help with anything?
No, thanks

Where did you hear about DuckDuckHack? (For first time contributors)
from hackathon sponsership

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
sinw

-----
IA Page: https://duck.co/ia/view/indian_languages_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@sinwar Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by sinwar** [comment]
@daxtheduck thanks!
waiting for positive response

**Comment by lxndio** [comment]
Looks good!
I would probably change too really minor things.
Firstly, a point at the end of the description would be good, I think.
Secondly, the sourceName should in my opinion better be "Languages with offical status in India - Wikipedia" so you know exactly what to expect when looking at the source link.

**Comment by moollaza** [comment]
@sinwar thanks a lot for contributing. We ask that you reach out before making any pull requests so we can approve your idea.

I'm a little reluctant to accept this but I think for now it's OK.

My reasoning is that I don't want to start a trend of duplicating information available as Wiki tables into cheat sheets, because ideally there should be a better way of accessing that information.

For example, it would be great if the tables were exposed through an API that could be used to search for tables, and return their info. Then we could create a table template that could re-create tables dynamically, instead of manually copying their info into Cheat Sheets.

I'll be discussing this with the team, so for now I think this is OK.

------

>Firstly, a point at the end of the description would be good, I think.

@lxndio we haven't been putting periods at the end of the Cheat Sheet descriptions. I'd prefer to not have them. Similarly, the subtitles for our other IAs never use periods.

>Secondly, the sourceName should in my opinion better be "Languages with offical status in India - Wikipedia" so you know exactly what to expect when looking at the source link.

Let's stick with `Wikipedia` the idea is to mention the name of the source, so the user has some idea of where the info is from and where they're going. If it becomes too wordy, you'll end up with a really long link, that won't look good on smaller screens.

**Comment by moollaza** [comment]
@sinwar by the way, can you please fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_cheat_sheet.md) and edit your initial comment. You should fill out the template for each new Instant Answer PR, to give the community and staff a better understanding of your Instant Answer. Once completed, we'll be sure to review your PR and give you some feedback.

Thanks!

**Comment by sinwar** [comment]
@moollaza I've updated my initial comment.
Waiting for positive feedback.
Thanks!

**Comment by gautamkrishnar** [comment]
Good:+1:

**Comment by sinwar** [comment]
@gautamkrishnar thanks, 
what about merge of PR?

**Comment by gautamkrishnar** [comment]
@moolaza @mintsoft will review the code and they will request you any changes if needed and will merge the PR

**Comment by sinwar** [comment]
okk @gautamkrishnar

File: share/goodie/cheat_sheets/json/languages-in-india.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                "val": "Hindi"
+            },
+            {
+                "key": "West Bengal",
+                "val": "Bengali, English"
+            }
+        ],
+        "Union Territories": [
+            {
+                "key": " Andaman & Nicobar Islands",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/languages-in-india.json]: Can you remove the spaces in the keys here? This entire section has them

**Comment by sinwar** [comment]
@mintsoft iIs PR perfect for merge now??

**Comment by mintsoft** [comment]
@sinwar it looks good to me, it's just down to @moollaza now :+1:

**Comment by sinwar** [comment]
@moollaza  checkout this PR and give feedback?

File: share/goodie/cheat_sheets/json/languages-in-india.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
+{
+    "id":"indian_languages_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/languages-in-india.json]: Currently we need the `id` and the filename to be the same. We'll need to change the filename to `indian-languages.json` and add `languages in india` as an alias

**Comment by moollaza** [comment]
@sinwar this LGTM :+1: -- there's a small fix that needs to be made the filename/alias but I can do that.

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by gautamkrishnar** [comment]
@sinwar congrats! :smiley:

**Comment by sinwar** [comment]
yo wlcm @moollaza 
Thanks @gautamkrishnar

**Comment by moollaza** [comment]
@sinwar congrats! It's live: https://duckduckgo.com/?q=indian+languages+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!
